pkgs/top-level/aliases.nix

```
@@ -1196,6 +1196,7 @@ mapAliases {
   microcodeAmd = microcode-amd; # Added 2024-09-08
   microcodeIntel = microcode-intel; # Added 2024-09-08
   micropad = throw "micropad has been removed, since it was unmaintained and blocked the Electron 27 removal."; # Added 2025-02-24
+  microsoft-edge = throw "microsoft-edge has been removed due to lack of maintenance";
```

```suggestion
  microsoft-edge = throw "microsoft-edge has been removed due to lack of maintenance in nixpkgs"; # Added 2025-05-19
```

Just to emphasize that upstream is fine and the package could possibly re-introduced if a set of maintainers/committers are willing to take care of it.
The date is helpful for the `maintainers/scripts/remove-old-aliases.py` maintenance script.